README.md

```diff
@@ -1,5 +1,5 @@
 
-![](https://raw.githubusercontent.com/solidusio/solidus/master/solidus.png)
+<img src="./logo.svg" width=350>
```

Comment:
Why don't we use markdown anymore?

Reply:
Regular HTML works in markdown but I'd prefer the markdown version too.

Reply:
AFAIK there isn't a way to manipulate size using Markdown.

Reply:
Yeah, I usually prefer Markdown over Markdown + HTML but I'm not totally against having HTML in the README. Is it worth it/possible to resave the SVG to a smaller size or will it simply enlarge itself by default of how the Markdown is rendered? Just thinking of other options if we really were set on not doing the HTML

Reply:
It's pretty huge, but if that's okay markdown works